### PR TITLE
[NOP][DOC] Updated the documentation pages of the installers. 

### PR DESCRIPTION
**Pull request by mwalzer**
…Including new OS versions, tested install instructions, corrected typos, added links and reflow.

### CODE REVIEW

File: doc/doxygen/install/install-linux.doxygen [hunk 1]
```diff
@@ -1,72 +1,65 @@
 /**
 
-  @page install_linux Building %OpenMS on GNU/Linux
+  @page install_linux Building OpenMS on GNU/Linux
```
Comment (timosachsenberg) [inline doc/doxygen/install/install-linux.doxygen]: Are you sure you don't need %OpenMS here?

File: doc/doxygen/install/install-linux.doxygen [hunk 2]
```diff
@@ -1,72 +1,65 @@
...
+    \code
+    # Assuming you are in ~/Development
+    git clone https://github.com/OpenMS/OpenMS.git
+    \endcode
 
-  \code
-  # Assuming you are in ~/Development
-  git clone https://github.com/OpenMS/OpenMS.git
-  \endcode
+  From here on, the instructions assume you have either an extracted tar.gz of a release and are inside its root, or checked out the development (git) version.
```
Comment (timosachsenberg) [inline doc/doxygen/install/install-linux.doxygen]: git/Git not consistently used

**Comment by timosachsenberg** [comment]
tx!

File: doc/doxygen/install/install-linux.doxygen (hunk 1 above)
Comment (mwalzer) [inline doc/doxygen/install/install-linux.doxygen]: yes, I am positive that I need it ;) - I think I figured it out now:
```@page``` you need it, ```@section``` you don't need it (it actually gets printed)

File: doc/doxygen/install/install-linux.doxygen (hunk 2 above)
Comment (mwalzer) [inline doc/doxygen/install/install-linux.doxygen]: tx, fixed the [G]

**Comment by timosachsenberg** [comment]
ah ok just saw your additions now

File: doc/doxygen/install/install-linux.doxygen (hunk 1 above)
Comment (mwalzer) [inline doc/doxygen/install/install-linux.doxygen]: yeah, the other way 'round of course. Dunno about comments, but it's removing the namespace link, but oxygen does so automatically in ```@page``` but not ```@section```. Also ```@page``` still shows %OpenMS in the browser tab and browser page title - at least my doxygen does so :(

**Pull request merged**